src/Compiler/Microsoft.AspNetCore.Razor.Language/src/TagHelperBinder.cs

```diff
@@ -46,43 +50,24 @@ public TagHelperBinder(string tagHelperPrefix, IReadOnlyList<TagHelperDescriptor
     /// <param name="parentIsTagHelper">Is the parent tag of the given <paramref name="tagName"/> tag a tag helper.</param>
     /// <returns><see cref="TagHelperDescriptor"/>s that apply to the given HTML tag criteria.
     /// Will return <c>null</c> if no <see cref="TagHelperDescriptor"/>s are a match.</returns>
-    public TagHelperBinding GetBinding(
+    public TagHelperBinding? GetBinding(
         string tagName,
         ImmutableArray<KeyValuePair<string, string>> attributes,
-        string parentTagName,
+        string? parentTagName,
         bool parentIsTagHelper)
     {
-        if (!string.IsNullOrEmpty(_tagHelperPrefix) &&
-            (tagName.Length <= _tagHelperPrefix.Length ||
-            !tagName.StartsWith(_tagHelperPrefix, StringComparison.OrdinalIgnoreCase)))
+        if (!TagHelperPrefix.IsNullOrEmpty() &&
+            (tagName.Length <= TagHelperPrefix.Length ||
+            !tagName.StartsWith(TagHelperPrefix, StringComparison.OrdinalIgnoreCase)))
```

Comment:
Nit: I know you didn't write this, but could you put an extra space in front of the `!` here to show it's inside the parens?

Reply:
Fixed